sdks/java/maven-archetypes/examples/generate-sources.sh

```diff
@@ -45,6 +45,20 @@ rsync -a --exclude cookbook --exclude complete --exclude '*IT.java'  \
     "${ARCHETYPE_ROOT}/src/test/java"				                 \
```

Comment:
For both rsync lines above, we should exclude the snippets directory from being copied over from the examples to the archetypes.

Reply:
done.